app/client/sagas/transferCardSagas.ts

```diff
@@ -0,0 +1,50 @@
+import { put, takeEvery, call, all } from "redux-saga/effects";
+import { message } from "antd";
+import { handleError } from "../utils";
+
+import {
+  EditTransferCardActionTypes,
+  EditTransferCardPayload
+} from "../reducers/transferCard/types";
+
+import { EditTransferCardAction } from "../reducers/transferCard/actions";
+
+import { LoadUserAction } from "../reducers/user/actions";
+
+import { editTransferCardAPI } from "../api/transferCardAPI";
+
+import { ActionWithPayload } from "../reduxUtils";
+
+function* editOrganisation(
+  action: ActionWithPayload<
+    EditTransferCardActionTypes.EDIT_TRANSFER_CARD_REQUEST,
+    EditTransferCardPayload
+  >
+) {
+  try {
+    const load_result = yield call(editTransferCardAPI, action.payload);
+
+    yield put(EditTransferCardAction.editTransferCardSuccess());
+
+    yield put(LoadUserAction.loadUserRequest({ path: action.payload.userId }));
```

Comment:
`action.payload.userId` is undefined, so this makes the FE call `/api/v1/user/?org=1` (with no user ID). This makes it load near-indefinitely on deployments with a lot of users (which is an out-of-scope problem)

Reply:
This is still an issue

Reply:
oh sorry i missinterpreted as leaving this for different pr!

Reply:
fixed now (in the edit component itself)